pkg/plugins/policies/meshtimeout/plugin/v1alpha1/plugin_test.go

```diff
@@ -541,6 +549,73 @@ var _ = Describe("MeshTimeout", func() {
 				},
 			},
 		},
+	}), Entry("route-level-timeouts", gatewayTestCase{
+		meshhttproutes: core_rules.GatewayRules{
+			ToRules: core_rules.GatewayToRules{
+				ByListenerAndHostname: map[core_rules.InboundListenerHostname]core_rules.Rules{
```

Comment:
if we could make this a golden file, that would be great

Reply:
IMO we don't have that many tests to introduce that logic. We could maybe have 2-3 and most of tests don't require this